docs/man/build-man.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
 
-pod2man -r "pg_activity 2.0.3" -d `date +%Y-%m-%d` -c "Command line tool for PostgreSQL server activity monitoring." pg_activity.pod > pg_activity.1;
+version=$(sed -e 's/__version__ = "\(.*\)"/\1/' ../../pgactivity/__init__.py)
+
+pod2man -r "pg_activity ${version}" -d `date +%Y-%m-%d` -c "Command line tool for PostgreSQL server activity monitoring." pg_activity.pod > pg_activity.1;
```

Comment:
How about calling `pg_activity --version`?

Reply:
why not, I choosed to take the version in the repo instead of the one installed to avoid mistakes but I am not deadset on it.

Reply:
On second thought, I think your approach is better as it indeed does not depend on the installation.